core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -54,7 +54,7 @@ import scala.util.control.ControlThrowable
  *       1. use new producer with following settings
  *            acks=all
  *            retries=max integer
- *            block.on.buffer.full=true
+ *            max.block.ms.config=max long
```

Comment:
I fixed that, but I  see another inconsistency in naming. Shouldn't `MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION` in org.apache.kafka.clients.producer.ProducerConfig
be named
`MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION_CONFIG` ?